update-hyprlock-layout.sh

```
@@ -0,0 +1,44 @@
+
+if grep -qF "$TARGET" "$CONF"; then
+    echo "Already configured: $TARGET"
+    exit 0
+fi
+
+cp "$CONF" "$BACKUP"
+echo "Backup created: $BACKUP"
+
+# Prefer replacing a plain "Enter Password" placeholder (with optional fingerprint markup)
+if grep -qE '^\s*placeholder_text\s*=' "$CONF"; then
+    sed -i 's|^\(\s*placeholder_text\s*=\s*\).*|\1Enter Password ($LAYOUT)|' "$CONF"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)